src/browser/workspace-api.ts

```diff
@@ -167,14 +167,14 @@ export class RestDevWorkspaceApi implements IDevWorkspaceApi {
   }
 
   private async waitForProjectToBeReady(namespace: string): Promise<void> {
-    let secondsAttempted = 0;
+    let millisecondsAttempted = 0;
     let request = await this.doesProjectExist(namespace);
-    while (secondsAttempted < this.projectInitRequestTimeoutMs && !request) {
+    while (millisecondsAttempted < this.projectInitRequestTimeoutMs && !request) {
       request = await this.doesProjectExist(namespace);
-      await delay(this.projectRequestDelay);
-      secondsAttempted += this.projectRequestDelay;
+      await delay(this.projectRequestDelayMs);
+      millisecondsAttempted += this.projectRequestDelayMs;
     }
-    if (secondsAttempted === this.projectInitRequestTimeoutMs) {
+    if (millisecondsAttempted === this.projectInitRequestTimeoutMs) {
```

Comment:
```suggestion
    if (millisecondsAttempted >= this.projectInitRequestTimeoutMs) {
```
^ operand may be wrong, just in case delay will be changed to 150 )